EffectPackages.ini

```diff
@@ -267,3 +267,12 @@ TextureInstallPath=.\reshade-shaders\Textures
 DownloadUrl=https://github.com/crosire/reshade-shaders/archive/legacy.zip
 RepositoryUrl=https://github.com/crosire/reshade-shaders/tree/legacy
 EffectFiles=AdaptiveSharpen.fx,AmbientLight.fx,Bloom.fx,Colourfulness.fx,Denoise.fx,DOF.fx,FakeMotionBlur.fx,FilmGrain2.fx,FilmicPass.fx,FineSharp.fx,GaussianBlur.fx,Glitch.fx,HighPassSharpen.fx,HQ4X.fx,HSLShift.fx,LevelsPlus.fx,LightDoF.fx,MagicBloom.fx,NightVision.fx,PPFX_Bloom.fx,PPFX_Godrays.fx,PPFX_SSDO.fx,ReflectiveBumpMapping.fx,StageDepth.fx,SurfaceBlur.fx
+
+[29]
+PackageName=Some Effects by Anastasia Bouwsma
+PackageDescription=Shaders made for interesting visual effects by Anastasia
+InstallPath=.\reshade-shaders\Shaderss\SomeEffects
```

Comment:
There is a `s` too much which snuck in there

Reply:
fixed that, my apologies